src/carma-streets/parse_kafka_logs.py

```diff
@@ -79,18 +86,29 @@ def parse_spat_to_csv(inputfile: Path, outputfile: Path):
     #write data of interest to csv which will be used to produce plots
     with open(outputfile, 'w', newline='') as write_obj:
         csv_writer = writer(write_obj)
-        csv_writer.writerow(['Created Time(ms)', 'Intersection Name', 'Intersection ID', 'Minute of the Year',
-                'Millisecond of the Minute', 'Intersection State'])            
+        csv_writer.writerow(['Created Time(ms)', 'Intersection Name', 'Intersection ID', 'Timestamp(ms)', 'Intersection State'])            
         skipped_messages = 0
+        #Need to get time since epoch of first day of year to use with moy and timestamp
+        #Our local timezone GMT-5 actually needs to be implemented as GMT+5 with pytz library
+        #documentation: https://stackoverflow.com/questions/54842491/printing-datetime-as-pytz-timezoneetc-gmt-5-yields-incorrect-result
+        
+        #Get year of test data from the first kafka create timestamp
+                           
+        test_year = str(datetime.datetime.fromtimestamp(int(spat_msgs[0].created_time)/1000)).split("-")[0]
+
+        naive = datetime.datetime(int(test_year), 1, 1, 0, 0, 0)
+        utc = pytz.utc
+        gmt5 = pytz.timezone('Etc/GMT+5')
+        first_day_epoch = utc.localize(naive).astimezone(gmt5).timestamp()*1000
```

Comment:
The `pytz` library appears to be deprecated. See their [maintenance mode notice](https://github.com/stub42/pytz/blob/master/src/README.rst#issues--limitations). Is there a specific reason we are using this library?

Reply:
I do not think so.  Will replace with tzdata as recommended.